test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperCheckboxTest.cs

```diff
@@ -649,6 +649,54 @@ public void CheckBoxFor_WithComplexExpressions_DoesNotUseValuesFromViewDataDicti
             Assert.Equal(expected, HtmlContentUtilities.HtmlContentToString(html));
         }
 
+        [Fact]
+        public void CheckboxHelpers_UsesSpecifiedExpression()
+        {
+            // Arrange
+            var helper = DefaultTemplatesUtilities.GetHtmlHelper();
+
+            // Act
+            var checkboxResult = helper.CheckBox("Property1");
```

Comment:
Similar to `RadioButtonHelpers_UsesSpecifiedExpression()`, must demonstrate what happens when `Property1 == "false"`, `Property1 == "FALSE"`, `Property1 = "true", and `Property1 == "Not a bool"`. Last case may already be handled (see a test using `Assert.Throws<FormatException>` below).

Suspect tests exist already covering `bool` properties. Double-check please.

I believe `bool` property